gammapy/data/pointing.py

```diff
@@ -25,6 +27,22 @@
 __all__ = ["FixedPointingInfo", "PointingInfo", "PointingMode"]
 
 
+def _check_coord(coord_or_frame, expected_frame, name):
```

Comment:
name is not used in this function. One could remove it, is not it?

Reply:
It is used in the error messages f-strings